lib/src/utils.dart

```diff
@@ -158,7 +164,7 @@ extension IntRange on int {
     } else if (this > n) {
       return rangeTo(n + 1);
     } else {
-      return const Iterable.empty();
+      return Iterable.empty();
```

Comment:
Is there a reason for removing this `const`?

Reply:
Nope, both this and the previous snippet you pointed out were due to a bad rebase onto master, apologies!